ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -641,4 +639,88 @@ def check_apps_running_on_selected_cluster(
         else:
             log.error(f"App {app} not found on cluster {cluster_name} on DR dashboard")
             return False
-    return True
+
+
+def verify_application_not_present_in_ui(acm_obj, workload_to_check, timeout=60):
+    """
+    Verify if application is not present in UI
+
+    Args:
+        acm_obj (AcmAddClusters): ACM Page Navigator Class
+        workload_to_check (str): Specify the workload to check,
+        timeout (int): timeout to wait for certain elements to be found on the ACM UI
+
+    Returns:
+        bool: True if the application is not present, false otherwise
+
+    """
+    if workload_to_check:
+        ocp_version = get_ocp_version()
+        acm_loc = locators[ocp_version]["acm_page"]
+        acm_obj.navigate_applications_page()
+        log.info("Click on search bar")
+        acm_obj.do_click(acm_loc["search-bar"])
+        log.info("Clear existing text from search bar if any")
+        acm_obj.do_clear(acm_loc["search-bar"])
+        log.info("Enter the workload to be searched")
+        acm_obj.do_send_keys(acm_loc["search-bar"], text=workload_to_check)
+        action_status = acm_obj.wait_until_expected_text_is_found(
+            locator=acm_loc["no-results-found"],
+            expected_text="No results found",
+            timeout=timeout,
+        )
+        if action_status:
+            fetch_status = acm_obj.get_element_text(locator=acm_loc["no-results-found"])
+            assert action_status, "Application present in UI"
+            log.info(
+                f"Successfully verified on ACM UI, Application Not Found, fetch status is {fetch_status}"
+            )
+            return True
+        else:
+            log.error("One or more matches found, Incorrect workload name")
```

Comment:
We should handle this situation by applying filters.
Eg. If 1 appset and 1 subscription based workload is deployed, UI will have 2 busybox-1 workloads but of different types. Returning False is not correct because the workloads are different even though names are same.

Reply:
@Shrivaibavi ^^

Reply:
The validation part is wholly modified according to the usecase, Hence marking this conversation as resolved.